matchbox/resource_profile.go

```diff
@@ -70,7 +76,7 @@ func resourceProfileCreate(d *schema.ResourceData, meta interface{}) error {
 
 	// Profile
 	name := d.Get("name").(string)
-	clcName, clc := containerLinuxConfig(d)
+	tmplName, tmpl := template(d)
```

Comment:
In the past, there have been cases its been useful to have both a container linux config and other config files (just one for now). Probably want to keep these separate.